akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpCookie.scala

```diff
@@ -8,10 +8,13 @@ import akka.http.impl.model.parser.CharacterClasses
 import akka.http.javadsl.model.headers
 import akka.parboiled2.CharPredicate
 import java.util.{ Optional, OptionalLong }
+
 import akka.http.scaladsl.model.DateTime
 import akka.http.impl.util._
 import akka.http.javadsl.{ model => jm }
 import akka.http.impl.util.JavaMapping.Implicits._
+import akka.http.javadsl.model.headers.SameSite
```

Comment:
So far, we created separate models for javadsl and scaladsl. It feels weird that you would have to import classes from `javadsl` to use it from Scala. That will probably mean duplicating the enums for Scala and Java and converting between them.